src/System.IO.Pipelines/System/Threading/Awaitable.cs

```diff
@@ -3,25 +3,25 @@
 
 using System.Runtime.CompilerServices;
 
-namespace System.IO.Pipelines
+namespace System.Threading
 {
```

Comment:
This is happening?

Reply:
Isn’t this supposed to be ValueAwaiter?

Reply:
@KrzysztofCwalina want's to keep generalized version in Pipelines if it's no going to BCL.